chaosprometheus/probes.py

```diff
@@ -23,6 +23,7 @@ def query(
     timeout: float = None,
     configuration: Configuration = None,
     secrets: Secrets = None,
+    verify_tls: bool = True,
```

Comment:
Could you put this before the `configuration` parameter? I tend, by convention, to keep the configuration and secrets as the last two arguments.